packages/cortex-react/src/service/Snackbar.tsx

```diff
@@ -0,0 +1,32 @@
+import { ExternalToast, toast } from 'sonner';
+import React from 'react';
+import { DefaultSnackSonner } from '../components';
+import { ISnackbar, TypeSnack } from './ISnackbar';
+
+export class Snackbar implements ISnackbar<ExternalToast> {
```

Comment:
Como essa implementação é focada em Sonner eu trocaria o nome para SnackbarSonner

Reply:
Então, tava com esse nome, mas eu pensei, o dev que for usar, não precisa saber em qual lib foi feita, ele só precisa saber que é um snack. Creio que não vamos ter implemenções de varias libs para o mesmo componente.

Mas agora que vai ter o provider, acho que já resolve isso tbm, e não vai ser um problema chamar de SnackbarSooner